diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/TrailingCommaRule.kt

```diff
@@ -101,9 +101,13 @@ class TrailingCommaRule(configRules: List<RulesConfig>) : DiktatRule(
     }
 
     private fun ASTNode.checkTrailingComma(config: Boolean) {
-        val shouldFix = this.siblings(true).toList().run {
-            !this.map { it.elementType }.contains(COMMA) && this.any { it.isWhiteSpaceWithNewline() || it.isPartOfComment() }
-        }
+        val noCommaInSiblings = siblings(true).toList()
```

Comment:
the only this I suggest is work with `toSeq` here

Reply:
do you mean `toSet`?

Reply:
changed to `toSet` + fixed `StringTemplateRuleFixTest`

Reply:
discussed offline -- it's not possible or will be unreadable